docs/elements/hx-popover/index.html

```diff
@@ -7,15 +7,17 @@
   elements/hx-div: <hx-div>
```

Comment:
Please update line 3 - `minver: 0.15.0` - to account for updated position logic.

Reply:
Let's hold off on updating `minver` for now. We need to figure out a different way of presenting version metadata, because `minver` no longer works well when we're modifying pre-existing stuff. We'd need something like `last updated` and `first introduced`, etc. As such, we'll have to update a lot of front matter (and jsdocs) to get things where we want them.

Reply:
Ok, we'll have to figure out a way of presenting `breaking changes` 🚨⚡️  in our online documentation.

So, that a consumer doesn't see `minver`, and think their code should work base on that version.